core/src/main/java/hudson/model/AbstractCIBase.java

```diff
@@ -94,6 +96,11 @@ protected void killComputer(Computer c) {
 
     /*package*/ final CopyOnWriteArraySet<String> disabledAdministrativeMonitors = new CopyOnWriteArraySet<String>();
 
+    @Restricted(NoExternalUse.class)
+    public CopyOnWriteArraySet<String> getDisabledAdministrativeMonitors(){
```

Comment:
```suggestion
    public @Nonnull CopyOnWriteArraySet<String> getDisabledAdministrativeMonitors(){
```

nice to have ;) (import already done)